nbproject/project.properties

```diff
@@ -13,46 +13,7 @@ auxiliary.org-netbeans-modules-apisupport-installer.os-solaris=false
-    ${project.au.gov.asd.tac.constellation.views.mapview}:\
-    ${project.au.gov.asd.tac.constellation.views.namedselection}:\
-    ${project.au.gov.asd.tac.constellation.views.pluginreporter}:\
-    ${project.au.gov.asd.tac.constellation.views.qualitycontrol}:\
-    ${project.au.gov.asd.tac.constellation.views.scatterplot}:\
-    ${project.au.gov.asd.tac.constellation.views.schemaview}:\
-    ${project.au.gov.asd.tac.constellation.views.scripting}:\
-    ${project.au.gov.asd.tac.constellation.views.tableview}:\
-    ${project.au.gov.asd.tac.constellation.views.timeline}:\
-    ${project.au.gov.asd.tac.constellation.views.webview}:\
-    ${project.au.gov.asd.tac.constellation.visual.opengl}:\
-    ${project.au.gov.asd.tac.constellation.webserver}
+modules=${project.au.gov.asd.tac.constellation.dependencies}\:${project.au.gov.asd.tac.constellation.functionality}\:${project.au.gov.asd.tac.constellation.graph}\:${project.au.gov.asd.tac.constellation.graph.file}\:${project.au.gov.asd.tac.constellation.graph.interaction}\:${project.au.gov.asd.tac.constellation.graph.node}\:${project.au.gov.asd.tac.constellation.graph.schema.analytic}\:${project.au.gov.asd.tac.constellation.graph.schema.visual}\:${project.au.gov.asd.tac.constellation.graph.utilities}\:${project.au.gov.asd.tac.constellation.graph.visual}\:${project.au.gov.asd.tac.constellation.plugins}\:${project.au.gov.asd.tac.constellation.plugins.algorithms}\:${project.au.gov.asd.tac.constellation.plugins.arrangements}\:${project.au.gov.asd.tac.constellation.plugins.importexport}\:${project.au.gov.asd.tac.constellation.preferences}\:${project.au.gov.asd.tac.constellation.security}\:${project.au.gov.asd.tac.constellation.testing}\:${project.au.gov.asd.tac.constellation.utilities}\:${project.au.gov.asd.tac.constellation.views}\:${project.au.gov.asd.tac.constellation.views.analyticview}\:${project.au.gov.asd.tac.constellation.views.attributecalculator}\:${project.au.gov.asd.tac.constellation.views.attributeeditor}\:${project.au.gov.asd.tac.constellation.views.conversationview}\:${project.au.gov.asd.tac.constellation.views.dataaccess}\:${project.au.gov.asd.tac.constellation.views.find}\:${project.au.gov.asd.tac.constellation.views.histogram}\:${project.au.gov.asd.tac.constellation.views.layers}\:${project.au.gov.asd.tac.constellation.views.mapview}\:${project.au.gov.asd.tac.constellation.views.namedselection}\:${project.au.gov.asd.tac.constellation.views.pluginreporter}\:${project.au.gov.asd.tac.constellation.views.qualitycontrol}\:${project.au.gov.asd.tac.constellation.views.scatterplot}\:${project.au.gov.asd.tac.constellation.views.schemaview}\:${project.au.gov.asd.tac.constellation.views.scripting}\:${project.au.gov.asd.tac.constellation.views.tableview}\:${project.au.gov.asd.tac.constellation.views.timeline}\:${project.au.gov.asd.tac.constellation.views.webview}\:${project.au.gov.asd.tac.constellation.visual.opengl}\:${project.au.gov.asd.tac.constellation.webserver}
```

Comment:
:cry: 
perhaps we can exclude this file from PRs if there isn't actually a change?